tests/framework/installer/ceph_manifests.go

```diff
@@ -3571,14 +3571,14 @@ spec:
           path: mon-endpoints`
 }
 
-func (m *CephManifestsMaster) GetBlockSnapshotClass(snapshotClassName, namespace, systemNamespace, reclaimPolicy string) string {
+func (m *CephManifestsMaster) GetBlockSnapshotClass(snapshotClassName, namespace, operatorNamespace, reclaimPolicy string) string {
```

Comment:
for better readability any plan to replace namespace with clusterNamespace?

Reply:
I think it's okay for the tests for now. I'd rather not introduce TOO many changes in this PR. Thoughts @travisn ?